src/main/java/frc/team3388/robot/SystemFactory.java

```diff
@@ -1,10 +1,18 @@
 package frc.team3388.robot;
 
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 import com.flash3388.flashlib.frc.robot.FrcRobotControl;
+import com.flash3388.flashlib.frc.robot.io.devices.FrcDoubleSolenoid;
+import com.flash3388.flashlib.frc.robot.io.devices.FrcSpeedController;
 import com.flash3388.flashlib.frc.robot.io.devices.SpeedControllers;
+import com.flash3388.flashlib.io.devices.DoubleSolenoid;
 import com.flash3388.flashlib.io.devices.SpeedController;
 import frc.team3388.robot.subsystems.ExampleSystem;
+import frc.team3388.robot.subsystems.IntakeSystem;
+
+import java.awt.Robot;
```

Comment:
Hmm what is this import? It's unused anyhow so remove.

You can do `ctrl+o` in Intellij to remove unused imports in a file.